primitives/allocator/src/freeing_bump.rs

```diff
@@ -60,8 +60,8 @@
 //!   fail.
 //!
 //! - Sizes of allocations are rounded up to the nearest order. That is, an allocation of 2,00001 MiB
-//!   will be put into the bucket of 4 MiB. Therefore, typically more than half of the space in allocation
-//!   will be wasted. This is more pronounced with larger allocation sizes.
+//!   will be put into the bucket of 4 MiB. Therefore, on average 25% of the space in allocation
+//!   will be wasted. This is more pronounced in absolute values with larger allocation sizes.
```

Comment:
```suggestion
//!   will be wasted. This is more pronounced (in terms of absolute amount of memory wasted) with larger allocation sizes.
```
Is this what you mean?

Reply:
Yes. I've rephrased a bit differently, cause I feel your proposal has too much emphasis on "memory wasted" and I don't think it's just, cause I don't think we can have an ideal allocator that doesn't have any waste at all.

Reply:
Nice rephrasing!